src/twisted/python/test/test_release.py

```diff
@@ -377,7 +377,14 @@ def inner():
             self.assertIsInstance(e, FailTest)
 
 
-@skipIf(not requireModule("pydoctor"), "Pydoctor is not present.")
+pydoctor = requireModule("pydoctor")
+
+
+@skipIf(pydoctor is None, "Pydoctor is not present.")
+@skipIf(
+    pydoctor is not None and pydoctor.__version__ < Version("pydoctor", 20, 12, 1),
+    "Pydoctor 20.12.1 or later is required.",
```

Comment:
do we need  this extra skip?

the setup.cfg asks for a newer pydoctor, so I think that we should be safe.

Reply:
We need it when running on Python 3.5, where pydoctor is excluded from the `dev_release` extra, but might still be installed anyway.